featuretools/tests/testing_utils/mock_ds.py

```diff
@@ -36,6 +37,9 @@ def make_ecommerce_entityset(with_integer_time_index=False):
          \\ /   .
           L     Log
     """
+    dask = pytest.importorskip("dask", reason="Dask not installed, skipping")
+    dask.config.set({"dataframe.convert-string": False})
```

Comment:
Setting this dask config allows the `object` dtype to be used by Woodwork, which allows LatLong columns to work and avoids an error with the Equal and NotEqual primitives, which were erroring if a `string[pyarrow]` column was called with a datetime column